TAF/testScenarios/functionalTest/V2-API/core-metadata/provisionwatcher/DELETE.robot

```diff
@@ -0,0 +1,24 @@
+*** Settings ***
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Suite Setup  Run Keywords  Setup Suite
+...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Remove Token
+Force Tags
+
+*** Variables ***
+${SUITE}          Core Metadata Provision Watcher DELETE Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-provisionwatcher-delete.log
+${api_version}    v2
+
+*** Test Cases ***
+ProWatcherDELETE001 - Delete provision watcher
+    Given Create Provision Watcher
+    When Delete Provision Watcher
+    Then Should Return Status Code "200"
+    And Response Time Should Be Less Than "1200"ms
```

Comment:
Should add `Should Return Content-Type "application/json"` to each testcases.

Reply:
Added.